tools/cibuildwheel_prep.sh

```diff
@@ -0,0 +1,5 @@
+# This is used in github actions when building the wheels for distribution.
```

Comment:
Is it worth putting a `set -e` at the beginning, to get a single error message instead of a bunch of them?

Reply:
That makes sense. Added